src/libstd/ffi/c_str.rs

```diff
@@ -326,6 +326,22 @@ impl fmt::Debug for CStr {
     }
 }
 
+#[stable(feature = "cstr_default", since = "1.9.0")]
```

Comment:
We just hit 1.8.0 release, so it becomes `since = 1.10.0` (if you could confirm @alexcrichton?).